app/data/repositories/UsersRepository.py

```diff
@@ -1,68 +1,110 @@
-from app.data import databaseConnection
-from app.data.models import User
+from typing import Optional, List
+
+from app.data import dbConn
+from app.data.models.Deck import Deck
+from app.data.models.User import User
+
+
+conn = dbConn.get()
+
+
+def create(name: str, email: str, password: str, birthday: str) -> bool:
+  
+    query = f"""INSERT INTO user (name, email, password, date_of_birth) 
```

Comment:
you still need triple "? in other files you use single "

Reply:
triple " for putting that string into 2 lines"